docs/sphinx/api/public_api.rst

```diff
@@ -129,6 +129,10 @@ Functions
 - :cpp:func:`hpx::ranges::set_intersection`
 - :cpp:func:`hpx::ranges::set_symmetric_difference`
 - :cpp:func:`hpx::ranges::set_union`
+- :cpp:func:`hpx::ranges::for_loop`
+- :cpp:func:`hpx::ranges::for_loop_strided`
+- :cpp:func:`hpx::ranges::for_loop_n`
+- :cpp:func:`hpx::ranges::for_loop_n_strided`
```

Comment:
Should we remove mentioning `ranges::for_loop_n` here as well?

Reply:
Yes, good catch!

Reply:
Done.